### PR TITLE
Updates for Alma8 (Software-5469)

### PR DESCRIPTION
**Pull request by jeff-takaki**
@brianhlin First attempt to add Alma8 to OS list. I still need to add the osg build container action but figured I would take a swing to see if this is looking like the right direction.

### CODE REVIEW

**Review by brianhlin — APPROVED** [review]
This looks pretty solid to me. Nice job!

**Comment by jeff-takaki** [comment]
@brianhlin Hey Brian, is this looking like it's in the ball park with the osg build container action?

File: .github/workflows/build-container.yml [hunk 1]
```diff
@@ -9,91 +9,70 @@ on:
...
+          IMAGE_LIST: ${{ steps.build-image.outputs.image-list}}
+        run: |
+          for registry in hub.opensciencegrid.org docker.io; do
+            IFS=,
+            for image in ${IMAGE_LIST}; do
+              fqin=hub.opensciencegrid.org/${image}
+              docker tag ${image} ${fqin}
+              docker push ${fqin}
+            done
+          done
```
Comment (brianhlin) [inline .github/workflows/build-container.yml]: All of this should be able to be replaced with a reference to the OSG build container action

**Review by brianhlin — COMMENTED** [review]
I think we can actually replace everything after and including the `build-image` step with 2 build container action steps (one for Harbor, one for DockerHub) that optionally pushes based on whether or not it's a merge to the main fork

The first step should have continue on error

**Comment by brianhlin** [comment]
@jtakaki-matc I've changed my mind: let's go with an additional matrix variable for the registry and only pushing if it's a commit to `master` on this fork. I think you should be able to make the registry a list of items containing a dict, something like:

```
- registry:
    - url: harbor URL
      user: harbor user
      pass: harbor pass
    - url: docker URL
    etc
```

**Comment by jeff-takaki** [comment]
@brianhlin Does this look like what you were thinking, or did you want me to try to make the push happen with the build-container-action?

**Comment by brianhlin** [comment]
Superseded by a move to Alma 9

**Pull request closed without merging**